src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:41853/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:41853/project/issues?id=sample&pullRequest=5884&issues=AWd6Di0eO_Y4hVnJ-pdk&open=AWd6Di0eO_Y4hVnJ-pdk)

[ISSUE_KEY]: AWd6Di0eO_Y4hVnJ-pdk